cmd/clusterlink/cmd/create/create_peer.go

```diff
@@ -71,13 +74,13 @@ func (o *PeerOptions) RequiredFlags() []string {
 
 func (o *PeerOptions) saveCertificate(cert *bootstrap.Certificate, outDirectory string) error {
 	// save certificate to file
-	err := os.WriteFile(filepath.Join(outDirectory, config.CertificateFileName), cert.RawCert(), 0o600)
+	err := os.WriteFile(filepath.Join(outDirectory, o.Name+".cert"), cert.RawCert(), 0o600)
 	if err != nil {
 		return err
 	}
 
 	// save private key to file
-	return os.WriteFile(filepath.Join(outDirectory, config.PrivateKeyFileName), cert.RawKey(), 0o600)
+	return os.WriteFile(filepath.Join(outDirectory, o.Name+".key"), cert.RawKey(), 0o600)
```

Comment:
Why did you change the file names to include the peer name?

Reply:
That is what I wanted to have <peer_name>.cert and <peer_name>.key that describes to whom this key belongs.